client/src/app/shared/models/poll/option.ts

```diff
@@ -1,8 +1,9 @@
 import { Id } from 'app/core/definitions/key-types';
-import { BaseDecimalModel } from '../base/base-decimal-model';
-import { HasMeetingId } from '../base/has-meeting-id';
+import { BaseModel } from '../base/base-model';
+
+export class Option extends BaseModel<Option> {
```

Comment:
I think you should use `BaseDecimalModel`: It allows you to give fields which values are automatically converted from decimal to number.